packages/nc-gui/lib/enums.ts

```diff
@@ -55,6 +55,10 @@ export enum Language {
   'zh-Hant' = '繁體中文',
 }
 
+export enum LanguageAlias {
+  zn_CN = 'zh-Hans',
```

Comment:
What about `zh-Hant`?

Reply:
Well, I can't assign zh_CH to two languages so I just picked the first one 🥴

Reply:
From the git history I found the following   

```
zh_Hans = '大陆简体',
zh_Hant = '臺灣正體',
```

```
zh_CN = '大陆简体',
zh_TW = '臺灣正體',
```

Reply:
```ts
export enum LanguageAlias {
  zn_CN = 'zh-Hans',
  zh_TW = 'zh-Hant',
}
```

So this?

Reply:
Yes 👍